src/main/java/org/opentripplanner/middleware/models/ApiUser.java

```diff
@@ -8,5 +8,26 @@
  * access otp-middleware's endpoints (as well as the geocoding and OTP endpoints).
  */
 public class ApiUser extends AbstractUser {
+    /** Holds the API keys assigned to the user. */
     public List<String> apiKeyIds = new ArrayList<>();
```

Comment:
When an ApiUser is created, the API keys should be created automatically for the various APIs. Please add a TODO for this in `ApiUserController#preCreateHook`.

Reply:
Noted in #39.